src/SarifCli/Rules/DoNotUseFriendlyNameAsRuleId.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Sarif.Cli.Rules
+{
+    public class DoNotUseFriendlyNameAsRuleId : SarifValidationSkimmerBase
+    {
+        public override string FullDescription => RuleResources.SV0001_DoNotUseFriendlyNameAsRuleIdDescription;
+
+        /// <summary>
+        /// SV0001
```

Comment:
Fixed.

---
In reply to: [77245086](https://github.com/Microsoft/sarif-sdk/pull/501#discussion_r77245086) [](ancestors = 77245086,77239380)